cyber/python/cyber_py3/test/BUILD

```diff
@@ -1,5 +1,8 @@
+load("@rules_python//python:defs.bzl", "py_test")
```

Comment:
It seems like a built-in function of bazel, just like cc_test.
Any reason to use the one from `rules_python`?

Reply:
> It seems like a built-in function of bazel, just like cc_test.
> Any reason to use the one from `rules_python`?

For higher bazel versions, `cc_test`, `py_test` were not builtin anymore,  and  the `load(...)` line was added by `buildifier`.